content/en/agent/logs/_index.md

```diff
@@ -22,17 +22,65 @@ further_reading:
 
 Log collection requires the Datadog Agent v6.0+. Older versions of the Agent do not include the `log collection` interface. If you are not using the Agent already, follow the [Agent installation instructions][1].
 
-Collecting logs is **disabled** by default in the Datadog Agent. Enable log collection in the Agent's [main configuration file][2] (`datadog.yaml`):
+## Activate log collection
+
+Collecting logs is **disabled** by default in the Datadog Agent.
+
+{{< tabs >}}
+{{% tab "(Recommended) HTTP compressed" %}}
```

Comment:
**Note**: Using long tab titles pushes tabs to an additional line. See the preview:
https://docs-staging.datadoghq.com/nils/agent-log-collection/agent/logs/?tab=httpuncompressed#activate-log-collection

Reply:
I saw but its fine on my end as the TCP is on the second line